databroker/tests/test_broker.py

```diff
@@ -592,7 +592,11 @@ def __call__(*args, **kwrags):
     assert not ev2['filled']['image']
     ev2_filled = db.fill_event(ev2, inplace=False)
     assert ev2_filled['filled']['image'] == ev2['data']['image']
-
+    ev_ret2 = db.fill_events([ev], h.descriptors, inplace=True)
+    ev_ret3 =  db.fill_events(ev_ret2, h.descriptors, inplace=True)
+    ev = next(ev_ret3)
+    assert ev['filled']['image'] == datum
+    assert ev['filled']['image']
```

Comment:
What is the purpose of this line?

Reply:
I guess that isn't needed now that we are not putting `True` in things.

Reply:
I'm just confused about 598 and 599. If 598 is true and `datum` is not false-y, then 599 we be true also.